sync.sh

```diff
@@ -2,9 +2,79 @@
 
 set -e
 
-# Read input arguments
+# Define Help function
+
+Help()
+{
+   # Display Help
+   echo "Tool to fill in belgian TST file from trade republic monthly report."
+   echo "This tool also creates a draft mail with the generated document and a qr code to pay the tax."
+   echo
+   echo "Syntax: sh sync.sh [-h|s|p|c] input_folder output_folder"
```

Comment:
```suggestion
   echo "Syntax: sh sync.sh [-h|s|p|c] input_folder [output_folder]"
```
The output folder is optional, right?

Reply:
Actually the input folder is also optional, should I remove the default value to make it mandatory ?